### PR TITLE
feat: first version of open api spec

### PR DESCRIPTION
**Pull request by fboucquez**
If you have time...

### CODE REVIEW

**Comment by fboucquez** [comment]
An example of how the open api docs could look like

https://fboucquez.github.io/statistics-service/openapi/index.html

**Comment by OlegMakarenko** [comment]
Looks great!

File: openapi/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+						// 		fontFamily: 'Noto Sans, sans-serif',
+						// 	},
+						// },
+						// breakpoints: {
+						// 	medium: '4500px',
+						// },
+						// spacing: {
+						// 	sectionVertical: 15,
+						// },
+					},
```
Comment (yilmazbahadir) [inline openapi/index.html:42]: Should we remove these theme settings if will not be used?

**Review by yilmazbahadir — COMMENTED** [review]
It looks good just put a tiny comment.

@OlegMakarenko  @AnthonyLaw should we merge this PR, if all is good?

**Comment by OlegMakarenko** [comment]
@fboucquez @yilmazbahadir
Looks good to me. I would better keep the schemas in separate files but it can be done later

**Comment by yilmazbahadir** [comment]
Thanks, @fboucquez shall we merge?
I'm working on this issue https://github.com/symbol/statistics-service/issues/95
Just added an API endpoint `/nodes/nodePublicKey/:nodePublicKey` , keen to get this merged and update both openapi.yml and wallets if possible.

**Comment by AnthonyLaw** [comment]
I add the new version, and SSL filter.
If all good, we can merge it :)

**Review by AnthonyLaw — COMMENTED** [review]
I add the new version, and SSL filter.
If all good, we can merge it :)

File: openapi/index.html (hunk 1 above)
Comment (rg911) [reply openapi/index.html:42]: the indentation doesn't look right to me

**Review by OlegMakarenko — APPROVED** [review]

**Pull request merged**